### ISSUE TITLE
corrupting project when more than one browser tab is open

### ISSUE THREAD

**Issue by sergpolly** [dekkerlab/cLIMS-docker#29]
@nandankita @betulakgol  let's discuss this critical issue here - with links to the code and such
keeping things open/transparent and documented would be great for us in the long run, and increase productivity

**Comment by sergpolly** [comment]
@nandankita  @betulakgol  is this one still relevant ?

**Comment by nandankita** [comment]
Yes, still not fixed yet.

**Comment by sergpolly** [comment]
@nandankita @betulakgol @Snehal1894 let's at least describe this bug - before we all forget  - it si something about project id on each browser tab ...
Minimal reproducible example would be the best of course

**Comment by sergpolly** [comment]
from @Snehal1894 words:
1. start an empty cLIMS  - something like that https://github.com/dekkerlab/cLIMS-docker/blob/master/README.md
2. create 2 projects - or use 2 existing ones (maybe) ?
3. open it in same browser with 2 tabs - 1 project per tab
4. open `experimentwizard` - for both projects - start entering data there  ...
5. whichever of the 2 - you save the last - will end up in the project that you opened "the latest" - the other one would remain empty ...

**Comment by nandankita** [comment]
The above description is correct. Just to re explain in another words:
1. Open clims
2. start experiment wizard for one project A.
3. Open another tab, go to another project B.
4. come back to experiment wizard tab, complete the form and save the experiment.
5. experiment will be added to the project B.

**Comment by Snehal1894** [comment]
From the code perspective,
The entries are added in the database, but are not linked correctly.
This is because, we are accessing the variables stored is 'session dictionary'.

So when we open a website/webpage, a session is started which is same for all the tabs in the a browser. And this session maintains a dictionary which can be used to pass the data.
So in the code, when 'request.session['ProjectId]= some value' is accessed, if I open two projects, this entry is overidden by the latest one.

**Comment by sergpolly** [comment]
so this is fixed now with the #65 ! Congrats @Snehal1894  and @nandankita !

**Comment by sergpolly** [comment]
@Snehal1894 tried other approaches:
> While trying to work with fixing the multiple tabs issue, I added this temporary hidden field. But we didn't use this approach. So I forgot to delete this field from the model. It is not needed.

```python
    class Meta:	    class Meta:
        model = Individual	        model = Individual
        exclude = ('individual_fields','userOwner','dcic_alias','update_dcic','temp_var')
```
